src/components/drag-and-drop/with-drag/with-drag.js

```diff
@@ -57,11 +58,15 @@ const ItemSource = {
   },
 
   beginDrag(props, monitor, component) {
+    // Disable Text selection in Safari to show correct cursor
+    BrowserHelper.getDocument().onselectstart = function() { return false; };
```

Comment:
Should we cache the existing value, and reset it on `endDrag` rather than setting it to `null`?